added discussed changes of the last sequencing round to the protokoll (#60)
StreetScienceCommunity/StreetScienceCommunity.github.io: _protocols/beer-dna-sequencing.md

## Requirements

### Needed consumables

MinION preparation

- Flow cell or Flongle

Library preparation

- 400ng DNA solved in 7.5µl nuclease-free water, extracted given the [DNA extraction protocol]({% link _protocols/beer-dna-extraction.md %})
- [Rapid Sequencing Kit](https://store.nanoporetech.com/sample-prep/rapid-sequencing-kit.html) including
    - 2.5 µl Fragmentation Mix (FRA)
    - 1 µl Rapid Adapter (RAP)
- Nuclease-free water (e.g. ThermoFisher, cat #AM9937)

    > "Normal" water contains nucleases which may degrade DNA

- 0.2 mL PCR tubes
- Ice

Sequencing

- Flow cell or Flongle
- [Rapid Sequencing Kit](https://store.nanoporetech.com/sample-prep/rapid-sequencing-kit.html) including
    - 30 µl Flush Tether (FLT)
    - 34 μl Sequencing Buffer (SQB)

>     -  µl Flush Buffer (FB)

Reviewer: How much quantity is needed there?

- P10 pipette and tips
- Vortex
- Centrifuge

## Install and prepare the Nanopore Software

> MinKNOW is the program needed to connect your computer with the small sequencing device MinION. The software has a graphical user interface for configuring and running the sequencing process. Furthermore, it has an integrated base-calling software to convert the raw Nanopore signals to the strings of nucleotides, in [fastq](https://en.wikipedia.org/wiki/FASTQ_format) file format.

1. Create an account on the Nanopore Community
2. Search for "MinION Software" on the [downloads page of Nanopore](https://community.nanoporetech.com/downloads)
3. Select the operating system of the computer
4. Follow the link and the installation instructions
5. Create an account to start the software

## Prepare the MinION

1. Attach the MinION device to the computer by connecting it to the USB port
2. Start the software

    2 possibilities:

    1. On Windows or Mac, click on the **MinION Software**
    2. On Linux
        - Open a terminal
        - Run `/opt/ui/MinKNOW`

3. Select the flow cell type as written on the flow cell package
4. Click on **"Check flow cell"** and then **"Start test"**

> <!--Add a screenshot-->

Reviewer: Do you have a screenshot of the software here?

    > Denaturation of DNA, i.e. the process by which double-stranded deoxyribonucleic acid unwinds and separates into single-stranded strands through the breaking of hydrophobic stacking attractions between the bases, takes place at 80°C. The primers are first annealed to target sequence at 30°C. 

6. Put the tube on ice until next step
7. Attach adapters by adding 1 µl of RAP into the tube

    > RAP contains adapters that are ligated to the DNA. They are needed for the sequencing to guide the DNA fragments through the nanopores

8. Mix gently by flicking the tube, and spin down
9. Incubate the reaction for 5 min at room temperature
10. Store the library on ice until ready to load

## Sequencing

1. Prepare the reagents
    1. Mix the SQB tube by vortexing, spin down and return to ice
    2. Mix the FB tube by vortexing, spin down and return to ice
    3. Spin down the FLT tube, mix by pipetting, and return to ice
2. Prepare loading port
    1. Open priming port on the flowcell to check for small bubbles
    2. Remove possible bubbles by taking 200 µl liquid from the port using the P1000 pipette
    3. Remove the liquid by turning the wheel of the pipet but only until 220-230 µl

>     Note: removing more than 30 µl will damage the pores in the array because they need to be covered by the buffer at all times

Reviewer: Given before it is written 200, I am not sure to understand how we can not remove more than 30 here

3. Prepare the **priming mix**

>     1. Add 30 µl of Flush Tether (FLT) directly to the Flush Buffer (FB) Eppi tube

Reviewer: Which quantity of FB is needed?

    2. Mix by pipetting

4. Load 800 μl of the priming mix into the flow cell via the priming port, avoiding introduction of air bubbles
5. Wait for 5 minutes

6. Prepare **library** for loading by mixing in a new tube
    - 34 μl Sequencing Buffer (SQB)
    - 25.5 μl Loading Beads (LB) (mix before adding)
    - 4.5 μl nuclease-free water
    - 11 μl DNA library

7. Load the library into the flow cell
    1. Gently lift the SpotON sample port cover to make the SpotON sample port accessible
    2. Load 200 μl of the priming mix into the flow cell via the **priming port (not the SpotON sample port)**, avoiding introduction of air bubbles
    3. Mix the prepared library gently by pipetting up and down prior to loading
    4. Add 75 μl of the library preparation to the flow cell via the SpotON sample port in a dropwise fashion
    
        Ensure each drop flows into the port before adding the next

    5. Gently replace the SpotON sample port cover
    6. Making sure the bung enters the SpotON port

>     7. Close the priming port and the MinION lid

Reviewer: Will the sequencing start automatically after? What happen in the software?
We should probably add also a screenshot of the software but also way to check that everything is going well

> - Fragmentation Mix (FRA)

Reviewer 2: ```suggestion
- Fragmentation Mix (FRA) from sequencing kit
  Random fragmentation of genomic DNAs is usually done by enzyme treatments
```

> - Rapid Adapter (RAP)

Reviewer 2: ```suggestion
- Rapid Adapter (RAP)
 Adapters are ligated to the DNA. They are needed for sequencing.
```

> - Nuclease-free water (e.g. ThermoFisher, cat #AM9937)

Reviewer 2: ```suggestion
- Nuclease-free water (e.g. ThermoFisher, cat #AM9937)
 "Normal" water contains nucleases which may degrade DNA. 
```

- 0.2 ml PCR tubes
- Ice

##### Needed material

> - Thermal cycler at 30° C and 80° C

Reviewer 2: ```suggestion
- Thermocycler at 30° C and 80° C
 A Thermocycler is a PCR machine to amplify DNA. Denaturation of DNA takes place at 80°C, annealing primers to target sequence at 30°C. 
```

> - Thermal cycler at 30° C and 80° C

Reviewer 2: ```suggestion
- Thermocycler at 30° C and 80° C
```

- Flush Buffer (FB)
- Loading Beads (LB)
- Nuclease-free water (e.g. ThermoFisher, cat #AM9937)
- 1.5 ml Eppendorf DNA 

> - MinIon diveice including Flow cell

Reviewer 2: ```suggestion
- MinIon device including Flow cell
```

- P1000 pipette and tips
- P20 pipette and tips
- P10 pipette and tips
- Vortex
- centrifuge

> #### Prepare substance of the priming mix

Reviewer 2: ```suggestion
#### Prepare priming mix
```

1. Mix the Sequencing Buffer (SQB) and Flush Buffer (FB) tubes by vortexing, spin down and return to ice
2. Spin down the Flush Tether (FLT) tube, mix by pipetting, and return to ice

#### Prepare loading port
1. Open priming port to check for small bubbles
2. To remove bubbles take some liquid from the port by use the P1000 pipette set to 200 µl

> 3. Remove the liquid by turning the weel but only until 220-230 µl

Reviewer 2: dont understand this. The wheel of the pipet?
